Formula/cgal.rb

```diff
@@ -11,57 +11,28 @@ class Cgal < Formula
     sha256 "861fcbdd2cea9fd8365e53e8ec7218f11c33b6b2fc1d0d732eeaa39c1b0343fd" => :yosemite
   end
 
-  option :cxx11
   option "with-qt", "Build ImageIO and Qt components of CGAL"
-  option "with-eigen", "Build with Eigen3 support"
-  option "with-lapack", "Build with LAPACK support"
 
   deprecated_option "imaging" => "with-qt"
   deprecated_option "with-imaging" => "with-qt"
-  deprecated_option "with-eigen3" => "with-eigen"
   deprecated_option "with-qt5" => "with-qt"
 
   depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on "eigen"
+  depends_on "gmp"
   depends_on "mpfr"
-
   depends_on "qt" => :optional
-  depends_on "eigen" => :optional
-
-  if build.cxx11?
-    depends_on "boost" => "c++11"
-    depends_on "gmp"   => "c++11"
-  else
-    depends_on "boost"
-    depends_on "gmp"
-  end
 
   def install
-    ENV.cxx11 if build.cxx11?
-
-    args = std_cmake_args + %W[
-      -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON
-      -DCMAKE_INSTALL_NAME_DIR=#{HOMEBREW_PREFIX}/lib
-    ]
-
-    if build.without? "qt"
-      args << "-DWITH_CGAL_Qt5=OFF" << "-DWITH_CGAL_ImageIO=OFF"
-    else
-      args << "-DWITH_CGAL_Qt5=ON" << "-DWITH_CGAL_ImageIO=ON"
-    end
-
-    if build.with? "eigen"
-      args << "-DWITH_Eigen3=ON"
-    else
-      args << "-DWITH_Eigen3=OFF"
-    end
-
-    if build.with? "lapack"
-      args << "-DWITH_LAPACK=ON"
-    else
-      args << "-DWITH_LAPACK=OFF"
-    end
-
-    system "cmake", ".", *args
+    qt = build.with?("qt") ? "ON" : "OFF"
+    system "cmake", ".", *std_cmake_args,
+                    "-DCMAKE_BUILD_WITH_INSTALL_RPATH=ON",
+                    "-DCMAKE_INSTALL_NAME_DIR=#{HOMEBREW_PREFIX}/lib",
+                    "-DWITH_CGAL_ImageIO=#{qt}",
+                    "-DWITH_CGAL_Qt5=#{qt}",
+                    "-DWITH_Eigen3=ON",
+                    "-DWITH_LAPACK=ON"
```

Comment:
I think the last two are useless. Also note that CGAL_ImageIO does not depend on qt and has zlib as optional dependency.

Reply:
@sloriot is there a minimum zlib? macOS ships with its own system zlib, which is usually good enough to enable such things.

Reply:
I don't think so. The one of the system should be fine.